Backpack/Dialog/Classes/BPKDialogIconView.m

```diff
@@ -49,19 +49,16 @@ - (CGSize)intrinsicContentSize {
     return [[self class] viewSize];
 }
 
-- (void)setIconBackgroundColor:(UIColor *_Nullable)iconBackgroundColor {
-    self.iconContainerView.backgroundColor = iconBackgroundColor;
+- (void)setIconDefinition:(BPKDialogIconDefinition *)iconDefinition {
```

Comment:
This argument should be `_Nullable`. If the user sets it to `nil` after it was not `nil` we need to ensure the correct thing happens